domains/Accounts/Controllers/AccountsLoginController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Domains\Accounts\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Accounts\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Hash;
+
+class AccountsLoginController extends Controller
+{
+
+    public function __invoke(Request $request): Response {
+
+        $this->validate($request, [
+            'email'    => ['required', 'string', 'email', 'max:255'],
+            'password' => ['required', 'string', 'min:6'],
+        ]);
+
+        $user = User::where('email', $request->email)->first();
```

Comment:
Maybe use `firstOrFail()` so that if it fails, we automatically get a `404` sent back. This also removes the need for the outer `else` below (lines 31-34)

Reply:
Yes, it makes sense.